### PR TITLE
[NETBEANS-54] Module Review css.prep

### LINKED ISSUE #34: Cleanup NetBeans license header for module css.prep
Review and cleanup the license header in css.prep

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in css.prep:
```
css.prep/src/org/netbeans/modules/css/prep/ui/resources/style.less
css.prep/src/org/netbeans/modules/css/prep/ui/resources/style.scss
css.prep/test/qa-functional/data/css_prep/public_html/cc.less
css.prep/test/qa-functional/data/css_prep/public_html/cc.scss
css.prep/test/qa-functional/data/css_prep/public_html/folder/part.scss
css.prep/test/qa-functional/data/css_prep/public_html/folder/partial.less
css.prep/test/qa-functional/data/css_prep/public_html/less_syntax.less
css.prep/test/qa-functional/data/css_prep/public_html/sass_syntax.scss
css.prep/test/unit/data/testFiles/test.less
css.prep/test/unit/data/testFiles/testIssue227676.scss
css.prep/test/unit/data/testProject/public_html/client1.scss
css.prep/test/unit/data/testProject/public_html/lib1.scss
css.prep/test/unit/data/testProject/public_html/test.less
css.prep/test/unit/data/testProject/public_html/test1.scss
css.prep/test/unit/data/testProject/public_html/test2.scss

```

### PR DESCRIPTION
**Pull request by tomazst**
Apache license added. Close #34

### CODE REVIEW

**Comment by junichi11** [comment]
Thanks for your help.

*.less and *.sass files don't support the `<#-- -->` comment. Then, CPWhereUsedQueryPluginTest fails if the license headers added to test data.(Because tests depend on offsets.) I don't know about qa-functional tests.

@emilianbold add `css.prep/test/unit/data/**` and `css.prep/test/qa-functional/data/**` to rat excludes?

**Comment by tomazst** [comment]
The extension is *.less and *.scss. But actually is a freemarker file. License is included with this line of code <#include "${project.licensePath}">. I guess you want it in a less and sass file. Hmmm. Before change, file css.prep/src/org/netbeans/modules/css/prep/ui/resources/style.less included license with a comment <#-- -->. I assumed this was the right way.

**Comment by junichi11** [comment]
I'm sorry I did not make it clear enough. Of course, template files are OK.
i.e. the following files are OK:
- css.prep/src/org/netbeans/modules/css/prep/ui/resources/style.less
- css.prep/src/org/netbeans/modules/css/prep/ui/resources/style.scss

But test files are not templates. Then, as I wrote above, unit tests fail if the license header is added to unit test file(e.g. css.prep/test/unit/data/testProject/public_html/lib1.scss). So, they should be added to rat excludes or unit tests must also be fixed.

**Comment by tomazst** [comment]
I changed headers in unit folder in to /* */.

File: css.prep/test/qa-functional/data/css_prep/public_html/cc.less [hunk 1]
```diff
@@ -1,3 +1,24 @@
+<#--
```
Comment (emilianbold) [inline css.prep/test/qa-functional/data/css_prep/public_html/cc.less:1]: I suspect this .less file does not support  `<#` comments

File: css.prep/test/qa-functional/data/css_prep/public_html/cc.scss [hunk 2]
```diff
@@ -1,3 +1,24 @@
+<#--
```
Comment (emilianbold) [inline css.prep/test/qa-functional/data/css_prep/public_html/cc.scss:1]: Same for this .scss file.

**Review by emilianbold — CHANGES_REQUESTED** [review]
It might be better to just exclude from Rat in build.xml the test files. They don't seem to be very complex at all, so there is nothing much to expressly protect.

**Comment by tomazst** [comment]
Closed by mistake

**Comment by tomazst** [comment]
I created new pull request #314

**Pull request closed without merging**